### DISCUSSION TITLE
The text in the canvas is broken

### DISCUSSION THREAD

**Discussion by JungYOsup** [wojtekmaj/react-pdf#1721] [Q&A]
When viewing a PDF on a new page(=Tab), the text in the canvas is broken.
Can you tell me why? And how to fix it would be appreciated.

```javascript 
 const newTab = window.open();
 newTab.document.documentElement.lang = "en";
 newTab.document.body.innerHTML = '<div id="root"></div>';

 Array.from(document.head.children).forEach((child) => {
          newTab.document.head.appendChild(child.cloneNode(true));
 });

 const root = ReactDOM.createRoot(
          newTab.document.getElementById("root") as HTMLElement
 );

 root.render(<PdfView file={blob} />);
```

<img width="1264" alt="image" src="https://github.com/wojtekmaj/react-pdf/assets/26478005/ee58c2c2-8286-4b8e-afde-efd1fbe1a8e1">

**Comment by wojtekmaj** [comment]
It's because fonts used by PDF are not available in new window context. I'm not aware of any workarounds.
